added an example for how to use options with the compiler.

robsaunders opened the pull request.

drewwells commented:
looks good, except file.sass is actually scss. The include is sass and gets translated correctly by libSass

robsaunders commented:
fixed, changed the filenames around to scss. That happened because I wrote it in sass and then realised your other examples were scss - in addition to the other issue I filed recently I thought maybe playing it safe and using scss would be the better option, but left the file names in. Whoops!
